docs/semgrep-code/semgrep-assistant-code.md

```diff
@@ -26,9 +26,8 @@ This document provides an overview of the following:
 * How to enable Semgrep Assistant
 * How to receive notifications or comments from Semgrep Assistant
 
-:::info Feature maturity 
-* This feature is in **public beta**.
-* You or your developers may encounter rough edges. This includes inaccurate recommendations or broken text. Please leave feedback on the PR comments from Semgrep Assistant to help the Semgrep team improve Semgrep Assistant.
+:::info Assistant content 
+Assistant content is generated by LLMs. Please review carefully and leave feedback on the PR comments from Semgrep Assistant to help the Semgrep team improve Semgrep Assistant.
```

Comment:
Not sure if keeping a little blurb in there would be helpful or not. Open to suggestions / deleting this

Reply:
I think the only thing I would change is add what LLM stands for

Reply:
resolved below, thanks Kaite!